AfterClass-io/afterclass.io-v2
# feat: password reset flow

## Context



Previously, the reset password flow was just a form input to enter a new password

This change introduces proper password reset with existing email. However, this will only work if supabase auth has already created the user. It will not work on v1 users.

## How to Test


1. go to `/`
2. click login button
3. click forgot password
4. enter email, click reset password, get redirected to `/account/auth/verify`
5. receive email
6. click link and get redirected to `/account/auth/reset-password`


## Checklist
